.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,19 @@
+#obtained from github market place
+# MIT License
+
+
```

Comment:
Looks like you're missing the `on` keyword: https://github.com/robertbegna/ubports-installer/actions